admin_site/system/models.py

```diff
@@ -1215,3 +1225,20 @@ def __str__(self):
 
     def __unicode__(self):
         return "{0}: {1}".format(self.problem.name, self.id)
+
+
+class ImageVersion(models.Model):
```

Comment:
This is a new model object with no associated migration -- please add one!

Reply:
It has been done. I had assign some stuff for the context object in order to highlight the targeted link. I'll push the changes later today.

Reply:
See my comment above about the numbering.